docs/fragments/cli-install-block.md

```diff
@@ -0,0 +1,35 @@
+<amplify-block-switcher>
+
+<amplify-block name="NPM">
+
+```bash
+npm install -g @aws-amplify/cli
+```
+
+</amplify-block>
+
+<amplify-block name="cURL (Mac and Linux)">
+
+```bash
+curl -sL https://aws-amplify.github.io/amplify-cli/install | bash && $SHELL
+```
+
+</amplify-block>
+
+<amplify-block name="cURL (Windows)">
+
+```bash
+curl -sL https://aws-amplify.github.io/amplify-cli/install-win -o install.cmd && install.cmd
+```
+
+</amplify-block>
+
+</amplify-block-switcher>
+
+**Note:**: Because we're installing the Amplify CLI globally, you might need to run the command above with `sudo` depending on your system policies.
```

Comment:
I would make this note only part of the npm install block. Don't want to encourage anyone to sudo curl a bash script!

Reply:
This note was there before, I'm not really sure why it was added but I assumed customers had issues before? Maybe @renebrandel or @swaminator can provide more context.

Reply:
Correct. Only have this for the NPM block

Reply:
@renebrandel I addressed this in the most recent commit. The `sudo` note is part of the NPM block and the ARM note part of the cURL (Mac and Linux) block